### DISCUSSION TITLE
providing own kafka/zookeeper images

### DISCUSSION THREAD

**Discussion by itpick** [strimzi/strimzi-kafka-operator#7182] [General]
Right now this is not possible with the helm chart due to a static suffix applied to every kafka image.

Are there some examples of providing your own kafka image and/or own zookeeper image that do not have a tag ending in "-kafka-3.2.0" ?

https://github.com/strimzi/strimzi-kafka-operator/blob/main/helm-charts/helm3/strimzi-kafka-operator/templates/_kafka_image_map.tpl#L16

**Comment by scholzj** [comment]
Not sure if it helps since you are not asking about it directly - but you can also specify the image in the Kafka CR directly in `.spec.kafka.image` / `.spec.zookeeper.image`.
